Source/Particles/MultiParticleContainer.cpp

```diff
@@ -662,12 +662,16 @@ MultiParticleContainer::doFieldIonization (int lev,
 }
 
 void
-MultiParticleContainer::doCoulombCollisions ()
+MultiParticleContainer::doCoulombCollisions ( Real cur_time )
 {
     WARPX_PROFILE("MPC::doCoulombCollisions");
 
     for( auto const& collision : allcollisions )
     {
+
+        const Real dt = WarpX::GetInstance().getdt(0);
+        if ( int(std::floor(cur_time/dt)) % collision->m_ndt != 0 ) continue;
```

Comment:
Would it be more robust to just place in the time step?

Do you already have an idea how we want to handle this in mesh-refined patches?

Reply:
@ax3l It should be easier and more robust to just modulo on the time step integer, but I don't see it is passed into `WarpX::OneStep_nosub (Real cur_time)`, but `cur_time` is already passed. I'm okay with either way.

I don't have an idea how to handle this in mesh-refined patches.
I feel like the collision module is independent to mesh refinement.

Reply:
Discussed this with @RemiLehe: `dt` on the refined patch will be automatically smaller so the current implementation will likely be solid.
I am not sure about the stability of the floating point division and floor for very large time steps, but we can cross that bridge when we come to it.